app/src/main/java/com/rlab/sejima/features/MUHeader.java

```diff
@@ -88,51 +103,158 @@ public MUHeader(Context context, AttributeSet attrs) {
         CharSequence s;
 
         // Deal with title's attributes
-        s = a.getString(R.styleable.MUHeader_title);
+        s = a.getString(R.styleable.MUHeader_header_title);
```

Comment:
MUHeader_title was looking good, why adding a second time header word?

Reply:
I had a conflict with the custom attribute `title` of the `MUTopBar` but it's fixed now.